.github/workflows/ci.yaml

```diff
@@ -60,7 +60,16 @@ jobs:
       actions: write  # Needed to manage GitHub Actions cache
 
   integration-test:
-    name: Integration test charm
+    strategy:
+      fail-fast: false
+      matrix:
+        juju-agent:
+          - 3.1.5
+          - 2.9.43
```

Comment:
It should be safe also to bump to `2.9.44` or there's some known issue?

Reply:
+1 to move 2.9.44